recipes/pyfastspm/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyfastspm" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d7b5c4c30fee2af38203290a6ac41c72408388b482bb98362f2a0973193efe6d
+  patches:
+    - temporary_setup_patch.diff  # remove on 0.11.1 release
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - packaging
+    - matplotlib-base
+    - h5py
+    - scipy
+    - numpy
+    - pillow
+    - tqdm
+    - ffmpeg
+    - mkl_fft
+    - scikit-image
+
+test:
+  imports:
+    - pyfastspm
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: http://faststm.gitlab.io/pyfast/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A python package for loading, processing and exporting scanning probe microscopy movies acquired with the FAST unit.'
+  doc_url: http://faststm.gitlab.io/pyfast/
```

Comment:
There is no `dev_url`. Is this intentional? I cannot access the project on GitLab. The source code on PyPI can be MIT licensed without the gitlab project being public, but maybe that's uninentional?

Reply:
> This seems to include https://fonts.google.com/specimen/Open+Sans which is Apache 2.0 licensed. Is this intentional?

Yes you are right, thanks, this slipped under our radar! I think we should include this both in the LICENSE file and in the `license_file` and `license_family` metadata. Not sure how though, maybe we should use [SPDX license expressions](https://spdx.github.io/spdx-spec/SPDX-license-expressions/) with something like `MIT AND Apache-2.0`

Reply:
We are planning to open the GitLab repo ASAP but yes, for the moment this is intentional since we need to iron out some issues before opening to the public.

Reply:
Yes, I believe that combining licenses with `AND`  is a possible solution here.

The alternative would be to add a dependency that provides a font that you can use. I am not sure about Open Sans but I know that matplotlib-base bundles a couple of fonts.

Reply:
This package has your font and also is an example how to have multiple licenses: https://github.com/conda-forge/open-fonts-feedstock/blob/master/recipe/meta.yaml